src/style/Style.scss

```diff
@@ -0,0 +1,16 @@
+@mixin truncate($truncation-boundary) {
```

Comment:
why file name start with capital it always start in smaller letter except components and this is not styles file its mixin type
and always add one line space between two mixins